exchanges/binance/cfutures_types.go

```diff
@@ -206,30 +206,27 @@ type BatchCancelOrderData struct {
 
 // FuturesOrderPlaceData stores futures order data
 type FuturesOrderPlaceData struct {
-	ClientOrderID string  `json:"clientOrderID"`
+	OrderID       int64   `json:"orderId"`
+	Symbol        string  `json:"symbol"`
+	Pair          string  `json:"pair"`
+	Status        string  `json:"status"`
+	ClientOrderID string  `json:"clientOrderId"`
+	Price         float64 `json:"price,string"`
+	AvgPrice      float64 `json:"avgPrice,string"`
+	ExecuteQty    float64 `json:"executeQty,string"`
 	CumQty        float64 `json:"cumQty,string"`
 	CumBase       float64 `json:"cumBase,string"`
-	ExecuteQty    float64 `json:"executeQty,string"`
-	OrderID       int64   `json:"orderID,string"`
-	AvgPrice      float64 `json:"avgPrice,string"`
-	OrigQty       float64 `json:"origQty,string"`
```

Comment:
Why has this been removed?

Reply:
Reordered.

Reply:
Actually OrigQty was deleted by mistake. Fixed.